convert.go

```diff
@@ -316,6 +344,16 @@ func convertVulnerabilities(vulns *[]Vulnerability, specVersion SpecVersion) {
 			vuln.Workaround = ""
 		}
 
+		if specVersion < SpecVersion1_6 {
+			if vuln.Credits != nil {
+				if vuln.Credits.Organizations != nil {
```

Comment:
Not too happy about all the nested `if`s, but wanted to avoid `&&`. Let me know what you think.